rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskController.java

```diff
@@ -405,11 +406,15 @@
   @PostMapping(path = RestEndpoints.URL_TASKS_ID_TRANSFER_WORKBASKET_ID)
   @Transactional(rollbackFor = Exception.class)
   public ResponseEntity<TaskRepresentationModel> transferTask(
-      @PathVariable String taskId, @PathVariable String workbasketId)
+      @PathVariable String taskId,
+      @PathVariable String workbasketId,
+      @RequestBody(required = false) Boolean setTransferFlag)
```

Comment:
did you test this manually with postman?

Reply:
@gitgoodjhe do you think we should create a test for this? I am not sure..

Reply:
No, unfortunately not. I'll do it now.